pkg/kubelet/eviction/helpers.go

```diff
@@ -203,7 +203,7 @@ func parseThresholdStatement(signal evictionapi.Signal, val string) (*evictionap
 		if percentage < 0 {
 			return nil, fmt.Errorf("eviction percentage threshold %v must be >= 0%%: %s", signal, val)
 		}
-		if percentage > 100 {
+		if percentage > 1 {
```

Comment:
Not seeing what this is fixing. Existing check looks fine.

Reply:
https://github.com/kubernetes/kubernetes/blob/eb2b71c727b87f5c9ea4fe4e69dac320972b8799/pkg/kubelet/eviction/helpers.go#L233-L240

I think the percentage here will be a float <= 1

Reply:
Exactly, when I develop a webhook about eviction configuration, I used this part of the code directly and found this problem. The upper limit more than 100% should be wrong.

Reply:
Thanks! Can we either add a comment or change the variable name to reflect that?

Reply:
for aprove @mrunalp